doc/v1/docs/superadmin_metric_templates.md

```diff
@@ -24,7 +24,65 @@ Mandatory fields in case of active metric template:
 * **Probe** - probe name and version. This field is autocomplete, user must choose from available probes;
 * **Type** - type (active or passive; active in this example);
 * **Probe executable** - executable file;
-* **Config** - those defined 5 config fields are mandatory for active metrics.
+* **Config** 
```

Comment:
Add short description for these 5 guys:
https://assets.nagios.com/downloads/nagioscore/docs/nagioscore/4/en/objectdefinitions.html#service